src/compas/robots/resources/github.py

```diff
@@ -52,11 +56,15 @@ def build_url(self, file):
         str
             The file's url.
         """
-        return '{}/{}/{}/{}/{}'.format(GithubPackageMeshLoader.HOST,
-                                       self.repository,
-                                       self.branch,
-                                       self.support_package,
-                                       file)
+        url_components = [
+            GithubPackageMeshLoader.HOST,
+            self.repository,
+            self.branch,
+            file
+        ]
+        if self.relative_path != '.':
+            url_components.insert(3, self.relative_path)
```

Comment:
The `insert(3, [..]` feels fragile. How about doing something like:
```python
relative_path_component = self.relative_path if self.relative_path != '.' else None
url_components = [
   GithubPackageMeshLoader.HOST,
   self.repository,
   self.branch,
   relative_path_component,
   file
]

return '/'.join(filter(None, url_components))
```

Reply:
oh, somehow I completely overlooked this comment before merging.  sorry.  i'll revisit this later.